cmd/deploy/remote.go

```diff
@@ -222,9 +249,14 @@ func getDeployFlags(opts *Options) []string {
 	return deployFlags
 }
 
-func getOriginalCWD(cwd, manifestPath string) string {
-	manifestPathDir := filepath.Dir(fmt.Sprintf("/%s", manifestPath))
-	return strings.TrimSuffix(cwd, manifestPathDir)
+// getOriginalCWD returns the original cwd
+func (rd *remoteDeployCommand) getOriginalCWD(manifestPath string) (string, error) {
+	cwd, err := rd.workingDirectoryCtrl.Get()
+	if err != nil {
+		return "", err
+	}
+	manifestPathDir := filepath.Dir(filepath.Clean(fmt.Sprintf("/%s", manifestPath)))
```

Comment:
same, use `filepath.Join()`

Reply:
This is using `filepath.Clean` which translates every separator into OS separator. We can't join a "" with a path because it will result in "./$manifestPath", and substraction wouldn't work

Reply:
oh yes, I read it wrong, apologies!